### PR TITLE
motorcycle profile for direction (copy from car)

### PR DESCRIPTION
**Pull request by nvcnvn**
## 👨‍💻 Changes proposed

Adding motorcycle profile (for now just copy everything from auto)

## 📄 Note to reviewers

I don't have a proper motorcycle icon, so I reuse bike icon

## 📷 Screenshots
Screenshot 2023-09-22 at 16 40 39

### CODE REVIEW

**Review by nilsnolde — CHANGES_REQUESTED** [review]
Copying (mostly) from auto should be fine, but we'd want
- at least `use_trails`: https://valhalla.github.io/valhalla/api/turn-by-turn/api-reference/#motorcycle-costing-options-beta
- definitely a motorcycle SVG

before merging.

File: src/Controls/settings-options.js [hunk 1]
```diff
@@ -688,6 +700,7 @@ export const settingsInit = {
   transit_start_end_max_distance: 2145,
   transit_transfer_max_distance: 800,
   disable_hierarchy_pruning: false,
+  use_trails: 1,
```
Comment (nilsnolde) [inline src/Controls/settings-options.js:703]: the default needs to be 0 as per docs

**Review by nilsnolde — CHANGES_REQUESTED** [review]
Just one more minor thing, then it's good to merge.

File: src/Controls/settings-options.js (hunk 1 above)
Comment (nvcnvn) [reply src/Controls/settings-options.js]: ooppps, not sure why I missed that.

**Comment by nilsnolde** [comment]
pity the PR workflow is broken, would love to see it before merging. not sure what's wrong there. seems the secret is not really taken by actions. I just re-pasted it but  still nothing.. smth must've changed in how secrets are referred to.

anyways, thanks for the changes @nvcnvn , looks safe to me to deploy regardless.

**Pull request merged**